debezium-core/src/main/java/io/debezium/transforms/TimezoneConverter.java

```diff
@@ -0,0 +1,388 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.OffsetTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.apache.kafka.connect.transforms.util.Requirements;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.data.Envelope;
+import io.debezium.time.MicroTimestamp;
+import io.debezium.time.NanoTimestamp;
+import io.debezium.time.Timestamp;
+import io.debezium.time.ZonedTime;
+import io.debezium.time.ZonedTimestamp;
+import io.debezium.util.BoundedConcurrentHashMap;
+
+public class TimezoneConverter<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TimezoneConverter.class);
+
+    private static final Field CONVERTED_TIMEZONE = Field.create("converted.timezone")
+            .withDisplayName("Converted Timezone")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.HIGH)
+            .withDescription("The timezone to convert to. " +
+                    "If not specified, the connector will use the default UTC timezone.")
+            .withDefault(ZoneOffset.UTC.toString());
```

Comment:
The original value should be in UTC so converting to UTC does not make sense, Maybe there should be no default value.

Reply:
Thinking about that a bit more, I'd really like to avoid using timezones completely and use timeshifts.
The parameter will accept value in format postiove or negative `hh:mm`. The value will be parsed and method `plus` on Instant/ZonedTimestamp etc. will be called to offset the time to appropriate value.